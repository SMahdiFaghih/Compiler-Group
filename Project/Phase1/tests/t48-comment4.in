
this is a /*comment*/ in between a line
