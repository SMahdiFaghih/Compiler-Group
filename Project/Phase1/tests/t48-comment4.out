this
T_ID is
T_ID a
T_ID in
T_ID between
T_ID a
T_ID line
